magento/pwa-studio
# Fixed PB Row Cypress tests

## Description

The page builder row cypress tests have been failing on the develop branch for a while now. They seem to be failing due to the following errors:

![image](https://user-images.githubusercontent.com/35203638/134234949-5d1649d2-2c37-4484-bc0f-6aab118167d6.png)

## Related Issue

Closes PWA-2118

## Acceptance

PB Row tests should pass

## Verification Steps

1. Run cypress tests using `yarn test:headless <Instance URL>` and tests should pass. (If you only wanna run PB row tests add the following option at the end of the docker run command in the run-tests.sh file `--spec ./src/tests/integrationTests/pageBuilder/row.spec.js`)

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
